README.md

```diff
@@ -68,6 +68,22 @@ dependencies {
 }
 ```
 
+### Compose versions
+
+`stripe-android` uses compose internally, and expects consumers to use a compatible compose version. Compatible versions listed below:
+
+<table>
+ <tr>
+  <td>20.22.0-20.31.0</td><td>Compose UI 1.4.x</td>
```

Comment:
This only shows Compose UI. We should do this for other Compose libraries. In the long term, let’s also figure out how we can automate this.

Reply:
Compose UI is the important one for compatibility. I'm not sure it's worth the effort to automate it at the moment. We've only changed it a few times in the last year.